migrations/202210131714933_add_schema_comments.up.sql

```diff
@@ -0,0 +1,281 @@
+COMMENT ON COLUMN public.github_pull_requests.deletions IS 'the number of deletions in the pull request';
+COMMENT ON COLUMN public.github_pull_requests.author_login IS 'login of the author of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.author_association IS 'author association to the repo';
+COMMENT ON COLUMN public.github_pull_requests.author_avatar_url IS 'URL to the avatar of the author of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.author_name IS 'name of the author of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.base_ref_oid IS 'the base ref associated with the pull request';
+COMMENT ON COLUMN public.github_pull_requests.base_ref_name IS 'the name of base ref associated with the pull request';
+COMMENT ON COLUMN public.github_pull_requests.base_repository_name IS 'the name of the base repo for the pull request';
+COMMENT ON COLUMN public.github_pull_requests.body IS 'body of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.changed_files IS 'the number of files changed/modified in the pull request';
+COMMENT ON COLUMN public.github_pull_requests.closed IS 'boolean to determine if the pull request is closed';
+COMMENT ON COLUMN public.github_pull_requests.closed_at IS 'timestamp of when the pull request was closed';
+COMMENT ON COLUMN public.github_pull_requests.comment_count IS 'the number of comments in the pull request';
+COMMENT ON COLUMN public.github_pull_requests.created_at IS 'timestamp of when the pull request was created';
+COMMENT ON COLUMN public.github_pull_requests.created_via_email IS 'boolean to determine if the pull request was created via email';
+COMMENT ON COLUMN public.github_pull_requests.database_id IS 'GitHub database_id of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.editor_login IS 'login of the editor of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.head_ref_oid IS 'the head ref associated with the pull request';
+COMMENT ON COLUMN public.github_pull_requests.head_ref_name IS 'the name of head ref associated with the pull request';
+COMMENT ON COLUMN public.github_pull_requests.head_repository_name IS 'the name of the head repo for the pull request';
+COMMENT ON COLUMN public.github_pull_requests.is_draft IS 'boolean to determine if the pull request is a draft';
+COMMENT ON COLUMN public.github_pull_requests.label_count IS 'number of labels associated to the pull request';
+COMMENT ON COLUMN public.github_pull_requests.last_edited_at IS 'timestamp of when the pull request was last edited';
+COMMENT ON COLUMN public.github_pull_requests.locked IS 'boolean to determine if the pull request is locked';
+COMMENT ON COLUMN public.github_pull_requests.maintainer_can_modify IS 'boolean to determine if a maintaner can modify the pull request';
+ALTER TABLE github_pull_requests
+DROP COLUMN IF EXISTS mantainer_can_modify;
```

Comment:
do we need to re-add this column in a subsequent migration?

Reply:
No the column exists with the correct spelling and is the comment right above this one.